subiquity/server/controllers/identity.py

```diff
@@ -72,13 +83,8 @@ def __init__(self, app):
 
     def load_autoinstall_data(self, data):
```

Comment:
suggestion: the documentation states that `append` and `override` are mutually exclusive, I think we should assert that here with an AutoinstallValidationError here.